app/src/main/java/com/example/pinkorange/vibrato/LiveWithSettings.java

```diff
@@ -111,48 +121,60 @@ protected void onCreate(Bundle savedInstanceState) {
             setSongDetails();
             bassBoost();
             loudnessEnhance();
+        } else {
+            setLiveDetails();
         }
     }
 
+    private void setLiveDetails() {
+        title.setText("Listening...");
```

Comment:
Can we change this to "Detecting audio..."? And can we put this in our `strings.xml` file for now so it's not hardcoded, then we can change it around?